.github/workflows/runtests.yml

```diff
@@ -149,6 +149,7 @@ jobs:
       - run: pip install mpmath gmpy2 matplotlib numpy scipy aesara ipython \
                          symengine cython llvmlite wurlitzer \
                          autowrap numexpr 'antlr4-python3-runtime==4.7.*'
+      - run: pip install numba
```

Comment:
This should just be added to the previous line

Reply:
can you please review now!!

Reply:
Please refrain from using exclamation marks, especially multiple. If it took you ten days to fix it, why the hurry now?